### ISSUE TITLE
Make "enter" staying in current pwd

### ISSUE THREAD

**Issue by aengelke** [debarshiray/fedora-toolbox#20]
It'd be cool if `fedora-toolbox enter` would stay in the current working directory (if available in the container, e.g. `~/**`). Currently, `enter` spawns a new shell with the cwd being the home directory, requiring to `cd` again.

**Comment by debarshiray** [comment]
Yes, I agree that we should do this. A few other people have asked for it too, and I personally feel the need for it while using `fedora-toolbox`.

**Comment by debarshiray** [comment]
One option is to add a `--workdir` option to `podman exec` and have it override the image's default working directory before executing the shell.

I learnt in `#podman` that `docker` already has such an option, so adding it will also help with compatibility.

**Comment by debarshiray** [comment]
`podman exec` [now has a --workdir option](https://github.com/containers/libpod/pull/2097) for specifying the working directory.
